yarn.lock

```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@7.0.0", "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```

Comment:
Non-essential for sure, especially for this PR, but these 2,000 lines of integrity hashes being removed: I'm guessing this is because we have different versions of yarn, in which case every PR I make with package.json changes will add these back, and every PR you make will remove them again. I'm on latest yarn, v1.16.0, if you aren't I would upgrade, `yarn global upgrade yarn`, and if you are, then that's interesting, I wonder why the yarn.lock differences.

Reply:
I'll take a look and upgrade if I'm not